docs/source/index.rst

```diff
@@ -269,6 +272,9 @@ conversion utilities for the following models:
 
 .. _bigtable:
```

Comment:
There is no need for that anchor now then. Maybe delete it and adapt references to that link elsewhere? I see five instances (this one excluded) with a quick search on the repo (one being the "end prompt" on check_copies that should use the line below, so "Supported frameworks" instead).